js/tests/friend-support_test.mjs

```diff
@@ -0,0 +1,125 @@
+import { once } from 'node:events'
+import * as cp from 'node:child_process'
+import { mkdir, writeFile, chmod } from 'fs/promises'
+import { tmpdir } from 'node:os'
```

Comment:
Imported but not used?

Reply:
I will remove the unused import statements